dotCMS/src/integration-test/java/com/dotcms/graphql/business/GraphqlAPITest.java

```diff
@@ -801,9 +801,33 @@ public void testGetSchema_GivenFailuresInRelationshipField_SchemaShouldStillGene
             assertNotNull(titleFieldDefinition);
         } finally {
             APILocator.getContentTypeAPI(APILocator.systemUser()).delete(contentType);
+            // restore normal RelationshipAPI for ContentAPIGraphQLTypesProvider
+            ContentAPIGraphQLTypesProvider.INSTANCE.setFieldGeneratorFactory(
+                    new GraphQLFieldGeneratorFactory());
         }
     }
 
+    @NotNull
+    private void setMockRelationshipAPI(Field relationshipField)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'relationshipField' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642169)